.devcontainer/scripts/onCreateCommand.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "Changing owner of workspace to vscode"
```

Comment:
I would recommend keeping this change out. Feel free to file a second a PR if this is something that needs addressing.

Reply:
You´re right I reverted it. My bad